Fst2List.cpp

```diff
@@ -2323,14 +2761,17 @@ const struct option_TS lopts_Fst2List[]= {
   {"input_encoding",required_argument_TS,NULL,'k'},
   {"korean",no_argument_TS,NULL,'K'},
   {"output_encoding",required_argument_TS,NULL,'q'},
+  {"make_dictionary",no_argument_TS,NULL,'P'},
   {"help",no_argument_TS,NULL,'h'},
+  {"binary dics",required_argument_TS,NULL,'D'},
   {NULL,no_argument_TS,NULL,0}
 };
 
 // FIXME(jhondoe) Use malloc to allocate chars' memory
 // FIXME(jhondoe) Full of possible memory leaks: aa.saveEntre, wordPtr2...
```

Comment:
I wrote this comment a few years ago. Could you check if the issues are now resolved?

Reply:
done